tests/e2e/MediaCo/portal.spec.js

```diff
@@ -100,30 +100,17 @@ test.describe('E2E test', () => {
     const sendToMgr = page.locator('label[data-test-id="C3B43E79AEC2D689F0CF97BD6AFB7DC4"]');
     await sendToMgr.check();
 
-    const currentCaseID = await page.locator('div[id="current-caseID"]').textContent();
-    const filePath = path.join(__dirname, '../../../assets/img/cableinfo.jpg');
+    const filePath = path.join(__dirname, '../../../../../assets/img/cableinfo.jpg');
 
     const attachmentID = await page.locator('div[id="attachment-ID"]').textContent();
     await page.setInputFiles(`#${attachmentID}`, filePath);
 
-    await Promise.all([
-      page.waitForResponse(
-        `${endpoints.serverConfig.infinityRestServerUrl}${
-          endpoints.serverConfig.appAlias ? `/app/${endpoints.serverConfig.appAlias}` : ''
-        }/api/application/v2/attachments/upload`
-      )
-    ]);
+    await expect(page.locator('CircularProgress')).not.toBeVisible();
+    await page.waitForTimeout(5000);
```

Comment:
Don't understand why you would want to add timeouts...seems like that would make the tests flakey.   We should discuss.

Reply:
These are the existing tests which we added in components repo.I don't see any failures with above code.

Reply:
Waiting on Promise was making tests flaky in test VM, so we decided to stick with the elements visible on screen thats why added circular progress check. Just to be sure the element is reflected on ui we are waiting for 5 seconds.